core/org.wso2.carbon.user.api/src/main/java/org/wso2/carbon/user/api/Tenant.java

```diff
@@ -222,6 +224,9 @@ public void setAdminUserId(String adminUserId) {
 
     public Map<String, String> getClaimsMap() {
 
+        if (claimsMap == null) {
+            return new HashMap<>();
```

Comment:
This also can lead some problems, if someone get the map and add an entry..

Better fix this in variable declaration level. So. it won't be null anyway.. 

    private Map<String, String> claimsMap = new HashMap<>();